.github/workflows/VeniceCI-CompatibilityTests.yml

```diff
@@ -0,0 +1,264 @@
+# GitHub Actions workflow for running compatibility tests: Avro, Alpini unit, Alpini functional tests, and Pulsar Venice integration tests
+
+name:  TestsDeCompatibilite
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+
+  AvroCompatibilityTests:
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [17]
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.jdk }}
+          distribution: 'temurin'
+          cache: 'gradle'
+      - shell: bash
+        run: |
+          git remote set-head origin --auto
+          git remote add upstream https://github.com/linkedin/venice
+          git fetch upstream
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: "-DmaxParallelForks=2 --parallel :internal:venice-avro-compatibility-test:test --continue"
+      - name: Package Build Artifacts
+        if: success() || failure()
+        shell: bash
+        run: |
+          mkdir ${{ github.job }}-artifacts
+          find . -path "**/build/reports/*" -or -path "**/build/test-results/*" > artifacts.list
+          rsync -R --files-from=artifacts.list . ${{ github.job }}-artifacts
+          tar -zcvf ${{ github.job }}-jdk${{ matrix.jdk }}-logs.tar.gz ${{ github.job }}-artifacts
+      - name: Upload Build Artifacts
+        if: success() || failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ github.job }}
+          path: ${{ github.job }}-jdk${{ matrix.jdk }}-logs.tar.gz
+          retention-days: 30
+
+  AlpiniUnitTests:
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [17]
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    outputs:
+      alpini_touched: ${{ steps.check_alpini_files_changed.outputs.alpini }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Checkout as many commits as needed for the diff
+          fetch-depth: 2
+      - name: Check if files have changed
+        uses: dorny/paths-filter@v2
+        id: check_alpini_files_changed
+        with:
+          filters: |
+            alpini:
+              - 'internal/alpini/**'
+      - uses: actions/checkout@v3
+        if: steps.check_alpini_files_changed.outputs.alpini == 'true'
+        with:
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        if: steps.check_alpini_files_changed.outputs.alpini == 'true'
+        with:
+          java-version: ${{ matrix.jdk }}
+          distribution: 'temurin'
+          cache: 'gradle'
+      # - name: Allow Deprecated TLS versions for Alpini tests
```

Comment:
Is this commented stuff needed?

Reply:
https://github.com/linkedin/venice/pull/244#discussion_r1132975861 - Same comment 😅

When I read through my reply, it doesn't make sense anymore as to how keeping this commented helps. It's okay to get rid of it. We can get it from git history now

Reply:
I see what I now meant in the thread. We have these two tests disabled:
```
@Test(enabled = false)
public void testLegacyClientTLS() throws Throwable {
...
}

@Test(enabled = false)
public void testLegacyServerTLS() throws Throwable {
...
}
```

Reply:
Let's take up this fight another day 😅